zerver/models.py

```diff
@@ -1927,11 +1927,23 @@ def get_bot_services(user_profile_id: str) -> List[Service]:
     return list(Service.objects.filter(user_profile__id=user_profile_id))
 
 def get_service_dicts_for_bots(user_profile_id: str) -> List[Dict[str, Any]]:
+    user_profile = get_user_profile_by_id(user_profile_id)
     services = get_bot_services(user_profile_id)
-    service_dicts = [{'base_url': service.base_url,
-                      'interface': service.interface,
-                      }
-                     for service in services]
+    service_dicts = []  # type: List[Dict[Text, Any]]
+    if user_profile.bot_type == UserProfile.OUTGOING_WEBHOOK_BOT:
+        service_dicts = [{'base_url': service.base_url,
+                          'interface': service.interface,
+                          }
+                         for service in services]
+    elif user_profile.bot_type == UserProfile.EMBEDDED_BOT:
+        from zerver.lib.bot_config import get_bot_config, ConfigError
```

Comment:
We should try to move this import to the top of the file.  If there are circular dependency issues, it might make sense to just move this entire function out of `models.py` and into `actions.py`, since it's starting to "outgrow" being just a thin model wrapper.